recipes/yaml-cpp/all/conanfile.py

```diff
@@ -64,6 +64,8 @@ def generate(self):
         if is_msvc(self):
             tc.variables["YAML_MSVC_SHARED_RT"] = not is_msvc_static_runtime(self)
             tc.preprocessor_definitions["_NOEXCEPT"] = "noexcept"
+        if self.conf.get("user.library.no_debug_suffix", default=False, check_type=bool):
```

Comment:
Do you have a suggestion for a different name to use? I am not sold on the conf name, but given previous push back against adding recipe options I figured using the same configuration would be preferable. Especially since usually one wants to remove it from all the libraries anyway.

Reply:
How would people feel about `core:no_debug_suffix` instead?

Reply:
I don't know why you are obsessed by these d postifx, but it's just something that conan can't guarantee since a d postifx can be hardcoded in upstream build files, and there is no reason to patch build files for such detail nobody cares.

Reply:
They break in some environments. But fair enough, I will simply keep them private from now on.